substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -3903,6 +3971,21 @@ impl<T: Config> Pallet<T> {
 			})
 			.unwrap_or_default()
 	}
+
+	/// Contribution of the member in the pool.
+	///
+	/// Includes balance that is unbonding currently.
+	pub fn api_member_total_balance(who: T::AccountId) -> BalanceOf<T> {
```

Comment:
It is confusing to me why and how a wallet should read this vs. look at the entire balance that that is held under the hold reason related to pools. How do these two differ? and in which way do you expect wallets to read user balances?

Reply:
> /// Includes balance that is unbonding currently.

Doesn't seem like enough explanation to me, as the unbonding balance is still present in the held amount. Could it be that this API is pending-slash-aware, whilst reading held balance is not?

Reply:
Will clarify this in the rust docs. Funds can be unbonded from `staking-pallet` but not from pool when unlock chunks are filled to its max capacity and an implicit unbonding is triggered by the pool.